.jenkinsci/test-launcher.groovy

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env groovy
+
+def chooseTestType() {
+	if (params.Merge_PR) {
+		if (env.NODE_NAME.contains('x86_64')) {
+			return "(module|integration|system|cmake|regression)*"	
```

Comment:
Also, better to write direct types in enums and operate on it.

Reply:
@muratovv fixed in the next commit